load-fhir-data.js

```diff
@@ -185,6 +185,26 @@ GC.get_data = function() {
                 }
             });
 
+            // Handle Father and Mother heights :
+
+            // Height of Father using LOINC if familyHistory is not available
```

Comment:
I'd just use a one line comment to address these sections.

// Handle father's and mother's heights using LOINC when Family History FHIR resource is not available.

Reply:
Fixed.